packages/deploy-script-support/src/getBundlerMaker.js

```diff
@@ -0,0 +1,38 @@
+// @ts-check
```

Comment:
Add a note that this should go away after we have bundlecaps, with a pointer to and from a relevant issue?